### PR TITLE
feat(Pipeline) Airflow BI Pipeline

### PR DESCRIPTION
**Pull request by clembac**

### CODE REVIEW

**Comment by clembac** [comment]
This MR brings the first version of Airflow - BI pipeline for Plastic Origin.
As now, airflow is hosted on a VM Azure, repository is deployed VM reboot.

For more details  on the pipeline cf to `/fillbidatabase/Pipelines`

To improve: 
- airflow deployment on terraform 
- separation of dev and prod airflow instances
- externalize job execution on K8 cluster rather than using  VM as a backend. For now, only `PostgresqlOperator`is used. 
- Test Dags  on CI.

File: Pipelines/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+This Airflow Pipeline must be used to compute metrics of `BI` schema in `PlasticOrigin` database
+
+## Requirements
+
+- [Apache-Airflow](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html), easier ton install localy using docker-compose ;) 
+
+- Creates connection to database (env `dev`or `prod`) 
+
+  ```bash
+  ./airflow.sh connections add [CONNECTIO] --conn-uri 'postgres://[USERNAME]@pgdb-plastico-dev:[PASSWORD]@pgdb-plastico-[ENV].postgres.database.azure.com:5432/plastico-[ENV]'
```
Comment (AntoineGirard) [inline Pipelines/README.md:12]:
```suggestion
  ./airflow.sh connections add [CONNECTIO] --conn-uri 'postgres://[USERNAME]@pgdb-plastico-[ENV]:[PASSWORD]@pgdb-plastico-[ENV].postgres.database.azure.com:5432/plastico-[ENV]'
```

File: Pipelines/scripts/clean_bi_temp_tables.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*
```
Comment (AntoineGirard) [inline Pipelines/scripts/clean_bi_temp_tables.sql]: Maybe we could remove this script?

Comment (clembac) [reply Pipelines/scripts/clean_bi_temp_tables.sql]: This script has been updated

**Pull request merged**